src/components/link/link.component.js

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { assign } from 'lodash';
+import { Link as RouterLink } from 'react-router';
+import Icon from '../icon';
+import { validProps } from '../../utils/ether';
+import Event from '../../utils/helpers/events';
+import tagComponent from '../../utils/helpers/tags';
+import LinkStyle from './link.style';
+import './link.scss';
+
+class Link extends React.Component {
+  static safeProps = ['onClick'];
+
+  constructor() {
+    super();
+    this.onKeyDown = this.onKeyDown.bind(this);
```

Comment:
Consider removing this and using a class method assignment for `onKeyDown` in the body of the `class`:

```
onKeyDown = (ev) => {
  // code
}
```

Reply:
Fixed